src/styles/cards.scss

```diff
@@ -0,0 +1,10 @@
+.cards {
```

Comment:
Please check 
![image](https://user-images.githubusercontent.com/54936044/236609996-1bae282b-5fc6-4586-ac0c-bd4a81905dd5.png)

Reply:
still not resolved

Reply:
didn't fix it